### PR TITLE
Update the public doc with the Multiple LNS REST APIs

### PR DESCRIPTION
**Pull request by ashhSAG93**

### CODE REVIEW

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 1]
```diff
@@ -1,37 +1,57 @@
 ---
 weight: 20
-title: Configuring ThingPark account credentials
+title: Configuring ThingPark account credentials and connections
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:3]: Configuring Multiple ThingPark account connections

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 2]
```diff
@@ -1,37 +1,57 @@
...
 layout: redirect
 ---
 
 
-Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure your ThingPark account details in the Administration application. In order to create new credentials or replace existing ones, go to the Administration application and select **Connectivity** in the **Settings** menu in the navigator.
+Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure your ThingPark account details in the Administration application. In order to create new connections or edit existing connections, go to the Administration application and select **Connectivity** in the **Settings** menu in the navigator. The **Connectivity** tab facilitates the creation of multiple Actility connections for use by different devices or switching between connections in a particular device.
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:8]: create new a connection or edit an existing connection.
remove "switching between connections in a particular device."

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 3]
```diff
@@ -1,37 +1,57 @@
...
+### <a name="create-new-connection">Creating a new connection</a>
 
-If you select **Connectivity** for the first time, you are asked to provide the Actility ThingPark URL and credentials which are used for LoRa device provisioning.
+If you select **Connectivity** for the first time, you are asked to create a connection. Click on Add Connection in the bottom left side of the screen. 
 
 Enter the following information:
 
+- **Name**: The name of the Actility connection being created. 
+- **Description**: The description of the Actility connection being created. 
 - **Actility ThingPark URL**: The Actility ThingPark URL may change based on the type of the Actility ThingPark account we use (for example Community or Enterprise edition). This is an optional field, and defaults to the value configured in the agent properties if nothing is entered.
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:18]: Remove "This is an optional field, and defaults to the value configured in the agent properties if nothing is entered."

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 4]
```diff
@@ -1,37 +1,57 @@
...
 - **Profile ID**: This depends on your ThingPark account and environment. If you are using, for example, the Dev1 ThingPark environment your profile ID will be "dev1-api". Multiple tenants can have the same profile ID.
+- **Route Application Server ID**: TLS security id between provider platform and agent. Application server id should be unique to route address. Leave empty to disable security.
+- **Route Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change application server key if Application server id is not updated. Leave empty to disable security.
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:21]: Optional Field.

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 5]
```diff
@@ -1,37 +1,57 @@
...
 - **Username**: Your ThingPark username.
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:22]: account username

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 6]
```diff
@@ -1,37 +1,57 @@
...
+The profile ID, username and password are used to retrieve an access token to send further requests to the ThingPark platform. It is possible to renew the access token by replacing the account credentials for a particular connection. 
 
 ![Setting provider credentials](/images/device-protocols/lora-actility/lora-admin-settings.png)
 
-Click **Save connectivity settings**. If you have entered the correct information, you see the message "Connectivity settings saved".
+Click **Save**. If you have entered the correct information, you see the message "Connection created".
 
-<a name="replace-credentials"></a>
-### Updating connectivity settings
+To add another connection, click on Add Connection in the bottom left side of the screen and then follow the steps mentioned above to create the connection.
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:32]: to create the new connection

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 7]
```diff
@@ -1,37 +1,57 @@
...
-Click **Save connectivity settings**. If you have entered the correct information, you see the message "Connectivity settings saved".
+Click **Save**. If you have entered the correct information, you see the message "Connection created".
 
-<a name="replace-credentials"></a>
-### Updating connectivity settings
+To add another connection, click on Add Connection in the bottom left side of the screen and then follow the steps mentioned above to create the connection. 
 
-In order to update your connectivity settings, go to the Administration and click **Update connectivity settings** in the **Settings** menu.
+<a name="update-credentials-in-a-connection"></a>
+### Updating connectivity settings for an existing connection
```
Comment (atulpandasag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:35]: Updating connection information for an existing connection

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 8]
```diff
@@ -35,20 +35,38 @@ After creating an "Associated user" with the proper group and profiles navigate
...
 
-The following information must be provided:
+The **Connectivity** tab facilitates the creation of multiple Sigfox connections for use by different devices or switching between connections in a particular device. 
+
+The following information must be provided in order to create a connection:
 
 - **Login:** The login token is located in the API access entry in the Sigfox Cloud.
 - **Password:** The password token is located in the API access entry in the Sigfox Cloud next to **Password**.
 - **Parent Group ID:** This ID is written in your URL when you are logged into your Sigfox account and you have selected the "Cumulocity" group. For example, "https://backend.sigfox.com/group/**9823ruj29j9d2j9828hd8**/info".
+- **Base URL:** This is the URL associated with the Sigfox Cloud account
```
Comment (atulpandasag) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:47]: URL that points

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**